Assets/Scripts/returnToLobby.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class returnToLobby : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        SceneManager.LoadScene("TutorialScene");
```

Comment:
If this is returnToLobby why does it load the TutorialScene?

Reply:
It's currently loading tutorial to test the flow so we can navigate between scenes, I didn't want to switch it to lobby before the lobby scene load issue is fixed.